rotkehlchen/utils/misc.py

```diff
@@ -273,15 +274,19 @@ def hexstr_to_int(value: str) -> int:
     return int_value
 
 
-def hex_or_bytes_to_int(value: Union[bytes, str], signed: bool = False) -> int:
+def hex_or_bytes_to_int(
+        value: Union[bytes, str],
+        signed: bool = False,
+        byteorder: Literal['little', 'big'] = 'big',
```

Comment:
ah yes .... the deposit contract uses little endian. They were pretty proud about it. If I recall it had something to do with the idea of targetting low-end processors and this has better performance there: https://github.com/ethereum/consensus-specs/pull/139

Reply:
Mmm, interesting! Thanks